src/actions/ast.js

```diff
@@ -117,7 +117,8 @@ function findBestMatch(symbols, tokenPos, token) {
     const overlaps =
       expression.location.start.line == line &&
       expression.location.start.column <= column &&
-      expression.location.end.column >= column;
+      expression.location.end.column >= column &&
+      !expression.computed;
```

Comment:
this is not related to this patch, but it would be nice to move `findBestMatch` to a utility and add some unit tests.

Reply:
Yeah! thanks for the confirm, thought about that ..but was wondering if we do it in another PR. i think it'll be cool to do here..